features/dnsresolver/netty/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -45,13 +52,19 @@
         <argument index="0" ref="eventForwarder"/>
         <argument index="1" ref="metricRegistry"/>
 
+        <property name="lookupEnabled" value="${lookup-enabled}"/>
         <property name="numContexts" value="${num-contexts}"/>
         <property name="nameservers" value="${nameservers}"/>
         <property name="queryTimeoutMillis" value="${query-timeout-millis}"/>
         <property name="minTtlSeconds" value="${min-ttl-seconds}"/>
         <property name="maxTtlSeconds" value="${min-ttl-seconds}"/>
         <property name="negativeTtlSeconds" value="${negative-ttl-seconds}"/>
 
+        <property name="breakerEnabled" value="${breaker-enabled}"/>
+        <property name="breakerFailureRateThreshold" value="${breaker-failureRateThreshold}"/>
```

Comment:
The words in the variables should be separated by dash instead of mixing dashes with snake-case. I.e. `breaker-failureRateThreshold` vs `breaker-failure-rate-threshold`.

Reply:
Done, a bit dash heavy now, open to alternatives.